test/torch/tensors/test_large_precision.py

```diff
@@ -218,3 +218,19 @@ def test_mod():
     lpt2 = x2.fix_prec(internal_type=internal_type, precision_fractional=precision_fractional)
     result = lpt1 % lpt2
     assert torch.all(torch.eq(expected, result.float_precision()))
+
+
+test_data = [
```

Comment:
I don't think is needed to create a variable just to use it in the decorator. Also, can you add tests with more decimal places (I assume this would work shouldn't it?) and negative numbers?

Reply:
Thanks for the reviews!

"I don't think is needed to create a variable just to use it in the decorator"
What would you suggest instead? Should I loop inside the test itself?

Reply:
You can move this array to the decorator itself. You'll have something like:

```python
 @pytest.mark.parametrize("x, expected", [(torch.tensor([1]), torch.tensor([1.0])),
    (torch.tensor([1.0]), torch.tensor([1.0])),
    (torch.tensor([2000.0, 1.0]), torch.tensor([2000.0, 1.0])),
    (torch.tensor([2000.0, 1]), torch.tensor([2000.0, 1.0])),
])
```

Reply:
Done